### DISCUSSION TITLE
Send Unity a byte[] array?

### DISCUSSION THREAD

**Discussion by brianmcclain** [jeffreylanters/react-unity-webgl#370] [Help Wanted!]
Is it possible to have my react app upload a file and then pass a byte[] array to Unity? I see there is a limitation on the type of data to send with UnityContext.

**Comment by jeffreylanters** [comment]
Hi. Sorry, this is a limitation by Unity. I've made it so that the types reflect the actual supported types by Unity. However, sending any kind of string is supported. I hope this helps you solve your problem.
